src/shared/adjusters/messages/en/increaseSize.json

```diff
@@ -6,5 +6,7 @@
     "additionalMagnifierAdjusters": "additional magnifier preferences",
     "additionalVisualAdjusters": "additional visual preferences",
     "moreMagnifierPreferences": "more magnifier preferences",
-    "lessMagnifierPreferences": "less magnifier preferences"
-}
\ No newline at end of file
+    "lessMagnifierPreferences": "less magnifier preferences",
```

Comment:
"less [magnifier preferences]" should be "fewer [...]", since preferences are countable.